src/widgets/_progressbars.scss

```
@@ -32,14 +32,40 @@ progressbar {
     }
 
     &.horizontal {
-        progress,
         trough {
             @extend %outset-background;
         }
 
         progress {
             margin-left: -1px;
             margin-right: -1px;
+
+            &:not(.pulse) {
+                animation: progress 2s ease infinite;
```

After playing with a bunch of values, this feels nicer to me. As a bonus, it's the same easing used elsewhere.

```suggestion
                animation: progress 1.5s $easing infinite;
```